app/js/views/service.jsx

```diff
@@ -19,18 +20,26 @@ export default class Service extends React.Component {
     });
   }
 
-  render() {
+  renderHeader() {
+    return super.renderHeader(
+      this.state.service && this.state.service.properties.name ?
+        this.state.service.properties.name :
+        'Unknown Service'
```

Comment:
But we had in ```if(!this.state.service)``` in ```renderBody``` I didn't check but I thought that there is a chance that render is called when we don't have service instance at all?